source/geh_common/src/geh_common/data_products/electricity_market_measurements_input/missing_measurements_log_metering_point_periods_v1.py

```diff
@@ -17,10 +17,14 @@
         # Metering point resolution: PT1H/PT15M
         T.StructField("resolution", T.StringType(), not nullable),
         #
-        # UTC time
+        # First time the metering point is connected or disconnected - or when resolution or grid area code has changed. UTC time
```

Comment:
Observation: In other contracts we tend to collect the periodization information in the schema docstring so it's neither scattered, nor repeated in individual columns.